pkg/version/version.go

```diff
@@ -14,68 +14,176 @@
 
 package version
 
-import "fmt"
+import (
+	"fmt"
+	"strings"
 
+	goversion "github.com/hashicorp/go-version"
+)
+
+// IstioOperatorVersionCompatibility is a mapping from an Istio operator version and the corresponding recommended and
+// supported versions of Istio.
+type IstioOperatorVersionCompatibility struct {
```

Comment:
VersionCompatibility or Compatibility to avoid stutter? IstioOperator seems redundant given the repository of this package :)

Reply:
Done.